logging/config.go

```diff
@@ -5,12 +5,35 @@ import (
 	"github.com/pkg/errors"
 	"go.uber.org/zap/zapcore"
 	"os"
+	"strings"
 	"time"
 )
 
 // Options define child loggers with their desired log level.
 type Options map[string]zapcore.Level
 
+// UnmarshalText implements encoding.TextUnmarshaler to allow Options to be parsed by env.
+func (o *Options) UnmarshalText(text []byte) error {
```

Comment:
It turns out the lib already supports `[]encoding.TextUnmarshaler`, so why not `map[T]encoding.TextUnmarshaler`?

* https://github.com/caarlos0/env/pull/323

Reply:
Thanks for trying to solve this issue upstream. I have updated the PR to reflect this potentially upcoming change in the method's signature, <https://github.com/Icinga/icinga-go-library/compare/7d7fd247cd9a69fddf46f237a6ae54ba475cb7e5..2ba2d8b19d9a9460f7f58cc2cd8d31561c03edf3>.

However, I am against waiting for this PR to get merged and a new env release to get drafted. If this will be supported in the future, the custom `encoding.TextUnmarshaler` can just be removed on our end.

Reply:
As written over at your env PR, it works fine. Thanks again.

However, I would still say to merge it in the current state, eventually removing the `encoding.TextUnmarshaler` implementation later on. We have no "influence" on the upstream, resulting in blocking this PR, resulting in blocking your PR, resulting in having less tests (your PR would be nice to have in #60) and also resulting in blocking the icinga-go-library release.